### ISSUE TITLE
Support Array for `sourceDir`

### ISSUE THREAD

**Issue by dmolsen** [pattern-lab/patternlab-php-core#32]
`sourceDir` (or `sourceDirs`) should be able to be defined as an array as a package creator may place assets in different locations.

**Comment by EvanLovely** [comment]
Been wanting this for awhile; it'd be really helpful! Wouldn't we want an array of `source/_patterns/` directories though? An array of `sourceDir`s could result in multiple `_data/data.json`s; how would that get handled, merging?

**Comment by dmolsen** [comment]
You're correct. I've updated the title of the ticket to reflect that. Just got focused on the `*Dir` options that _are_ available and `patternDir` isn't among them. Guess it needs to be :)

**Comment by sghoweri** [comment]
@EvanLovely need any help tackling this?

**Comment by EvanLovely** [comment]
@sghoweri Sure! I feel like #35 is a prerequisite though.

**Comment by sghoweri** [comment]
@EvanLovely Ohhhh sorry - missed that. Need help on that one then?

**Comment by EvanLovely** [comment]
@sghoweri Would love it! Got a PR started: #152
